service/worker/scanner/taskqueue/db.go

```diff
@@ -59,8 +60,9 @@ func (s *Scavenger) getTasks(key *p.TaskQueueKey, batchSize int) (*p.GetTasksRes
 			NamespaceID: key.NamespaceID,
 			TaskQueue:   key.Name,
 			TaskType:    key.TaskType,
-			ReadLevel:   -1, // get the first N tasks sorted by taskID
-			BatchSize:   batchSize,
+			MinTaskID:   -1, // get the first N tasks sorted by taskID
+			MaxTaskID:   math.MaxInt64,
```

Comment:
This could cause too many tombstone issue?

Reply:
the number of batch size will be the upper limit